modin/pandas/test/utils.py

```diff
@@ -128,19 +128,27 @@
         ]
         for i in range(NCOLS)
     },
-    "subset_duplicates": {
-        "col{}".format(i): [
-            i if j % 7 == 0 and i in [1, 3, 7] else x
-            for j, x in enumerate(range(NROWS))
-        ]
-        for i in range(NCOLS)
-    },
     "has_name_column": {
         "name": ["one", "two", "two", "three"],
         "col1": [1, 2, 2, 3],
         "col3": [10, 20, 20, 3],
         "col7": [100, 201, 200, 300],
     },
+    "float_nan": {
+        "col{}".format(int((i - NCOLS / 2) % NCOLS + 1)): [
+            np.NaN if (j % random_state.randint(1, 5) != 0) else x
+            for j, x in enumerate(
+                random_state.uniform(RAND_LOW, RAND_HIGH, size=(NROWS))
+            )
+        ]
+        for i in range(NCOLS)
+    },
```

Comment:
probably you can copy `float_nan_data` from `test_data` dataset

Reply:
@anmyachev `float_nan_data` doesn't contain duplicated rows with all `NaN`'s

Reply:
ok, what about manually change first two rows in `float_nan_data` to your values?

Reply:
I use `float_nan` data in several places and duplicate rows are distributed throughout all data. I think that manually change of  the same data in several tests isn't good approach. Please, correct me if I'm wrong.

Reply:
You are absolutely right, but I mean that this can be done in utils.py scope.

Something like that:
```python
for key in test_data["float_nan_data"]:
    test_data["float_nan_data"][key][0] = np.NaN
    test_data["float_nan_data"][key][1] = np.NaN
```

After this just copy `test_data["float_nan_data"]` into "duplicated" dataset.

Reply:
This will diversify the `test_data["float_nan_data"`] and save us from generating new data. What do you think?

Reply:
Well, I can do it, but in this case I will need to merge `test_data["float_nan_data"]` in `test_data_with_duplicates` (which I use for tests) when I will make data for test.

Reply:
You speak about this: `test_data_with_duplicates["float_nan"] = test_data["float_nan_data"]`?

Any problem with this?

Reply:
I used your approach, check please.

Reply:
Thanks @prutskov! It's OK for me.